framework/include/arastorage/arastorage.h

```diff
@@ -66,6 +66,7 @@
  ****************************************************************************/
 #include <stdbool.h>
 #include <stdio.h>
+#include <float.h>
 /****************************************************************************
```

Comment:
@jeongarmy Hello! As far as I understand there are two variants exactly what "float.h" will be included by this directive depending on whether CONFIG_ARCH_FLOAT_H is defined. 

In the case ARCH_FLOAT_H option enabled, include/tinyara/float.h would be copied into the include/float.h.
Otherwise, if ARCH_FLOAT_H is disabled, the the float.h from toolchain will be included.

So by making "#include <float.h>" we make it possible for user to choose what header to include by enabling/disabling ARCH_FLOAT_H option. If we include directly tinyara/float.h, then ARCH_FLOAT_H would have no effect, which seems to be not right.